ci/stage-test-sgx.jenkinsfile

```diff
@@ -30,6 +30,12 @@ stage('test-sgx') {
         timeout(time: 5, unit: 'MINUTES') {
             sh '''
                 cd Examples/python
+                os_release_id=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
+                if [ "${os_release_id}" == 'rhel' ]
+                then
+                    sed -i 's/sgx.enclave_size = "512M"/sgx.enclave_size = "64G"/g' python.manifest.template
```

Comment:
We will be having 2 systems for RHEL only one of them will be server and 2nd will not have enough memory, it will be having only 8GB. Modify this condition to suit both of the systems